.github/workflows/release.yml

```
@@ -20,21 +20,24 @@ jobs:
           fetch-depth: 0
           token: ${{ secrets.SEEK_OSS_CI_GITHUB_TOKEN }}
 
+      - name: Set up pnpm
+        uses: pnpm/action-setup@v3
```

This should be v4 too.
```suggestion
        uses: pnpm/action-setup@v4
```